packages/channels/src/messenger/README.md

```diff
@@ -1,29 +1,31 @@
+# Messenger (v1.0.0)
+
 ### Sending
 
-| Channels | Messenger |
-| -------- | :-------: |
-| Text     |    ✅     |
-| Image    |    ✅     |
-| Choice   |    ✅     |
-| Dropdown |    ✅     |
-| Card     |    ✅     |
-| Carousel |    ✅     |
-| File     |    ✅     |
-| Audio    |    ✅     |
-| Video    |    ✅     |
-| Location |    ✅     |
+| Channels | Messenger | Details                    |
+| -------- | :-------: | :------------------------- |
+| Text     |    ✅     |                            |
+| Image    |    ✅     |                            |
+| Choice   |    ✅     |                            |
+| Dropdown |    ✅     |                            |
+| Card     |    ✅     |                            |
+| Carousel |    ✅     |                            |
+| File     |    ✅     | File sent as text link     |
+| Audio    |    ✅     | Audio sent as text link    |
+| Video    |    ✅     | Video sent as text link    |
+| Location |    ✅     | Location sent as text link |
```

Comment:
We should say it's a google maps link here and for all the other channels that down render locations the same way